abstraction-textbox-class/src/main/java/TextBox.java

```diff
@@ -0,0 +1,39 @@
+package main.java;
+
+public class TextBox {
+    private String value;
+    private boolean enabled;
+    int maxLength = 15;
```

Comment:
I asked why the field is private. does not necessarily mean that you have to remove it

Reply:
tama naman. but initially private static yung ginawa mo, wala naman ako naturo na private static so far. so bakit mo sya ginawang private na static?